pandas/_libs/window/aggregations.pyx

```diff
@@ -122,9 +122,9 @@ def roll_sum(const float64_t[:] values, ndarray[int64_t] start,
              ndarray[int64_t] end, int64_t minp) -> np.ndarray:
     cdef:
         Py_ssize_t i, j
-        float64_t sum_x = 0, compensation_add = 0, compensation_remove = 0
+        float64_t sum_x, compensation_add, compensation_remove
```

Comment:
you need to leave these no?

Reply:
These get initialized under the `if i == 0` condition; they are not used before that point.

Reply:
ok does the compiler complain on building?

Reply:
No complaints from cythonize or gcc.